go-controller/pkg/ovn/policy_common.go

```diff
@@ -165,6 +168,201 @@ func (gp *gressPolicy) delAddressSet(hashedAddressSet string) (string, string, b
 	return oldL3Match, gp.getL3MatchFromAddressSet(), true
 }
 
+func (oc *Controller) handlePeerPodSelectorAddUpdate(np *namespacePolicy,
```

Comment:
docblock please!

Reply:
@squeed it's copy & paste, there wasn't one before.

Reply:
for sure, but at this point, you understand this function. if you could capture that, that'd be good.